### DISCUSSION TITLE
MicroStream CE for enterprise development

### DISCUSSION THREAD

**Discussion by klysov** [microstream-one/microstream#314] [Q&A] [answered]
Can I use MicroStream CE for enterprise development for free?
Are there any restrictions?

**Comment by hg-ms** [comment, marked as answer]
The CE edition is released under the [Eclipse Public License 2.0 license](https://github.com/microstream-one/microstream/blob/master/LICENSE). Therefore, you are free to use it in commercial projects according to that license for free.

The versions differ in the supplied support and data storage connectors. Please have a look at https://microstream.one/editions/ for details.
